.travis.yml

```diff
@@ -27,4 +27,4 @@ install:
   - python setup.py install
 
 script:
-  - python -m unittest discover --start-directory magpie/tests
+  - python -m unittest discover
```

Comment:
On INSPIRE we now use the much more powerful `py.test` and tests are kept outside of the code base. Maybe this is some further improvement (not urgent) to make the codebase even more clean.